CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ For details about compatibility between different releases, see the **Commitment
 - Experimental support for BasicStation GPS timestamps which use the wrong precision (milliseconds instead of microseconds).
   - The Gateway Server will attempt to determine the correct GPS timestamp from the provided `gpstime` based on the time at which the upstream message has been received.
   - This workaround will be available until the related gateway vendors will release patches for this issue.
+- Support for Azure Blob Storage. Only authentication via MSI is supported
```

Comment:
I would call this `experimental support`, given that we're not sure yet if this is the final form of the configuration. Also dot at the endl.